recipes/.gitignore

```diff
@@ -0,0 +1,5 @@
+# conda smithy ci-skeleton start
+*.pyc
+
+build_artifacts
+# conda smithy ci-skeleton end
```

Comment:
Please remove this and only change things in your recipe.

Reply:
Done.

Reply:
Thanks. I don't review python-c but let me assign them to have this moved along.

Reply:
Thanks! Do I need to ping the c-cpp team directly, or will they see the assigned label?